services/credential-server/src/agent.ts

```diff
@@ -161,35 +129,72 @@ class Agent {
       await this.signifyApi.deleteNotification(notif.i);
     }
   }
-  async initKeri() {
+  async initKeri(): Promise<void> {
     this.onNotificationKeriStateChanged();
     /* eslint-disable no-console */
-
     // Issuer
-    const existedIdentifier = await this.signifyApiIssuer
-      .getIdentifierByName(Agent.ISSUER_AID_NAME)
-      .catch(() => null);
-    if (existedIdentifier) return existedIdentifier;
-
     await this.signifyApiIssuer
       .createIdentifier(Agent.ISSUER_AID_NAME)
       .catch((e) => console.error(e));
     this.keriIssuerRegistryRegk = await this.signifyApiIssuer
       .createRegistry(Agent.ISSUER_AID_NAME)
       .catch((e) => console.error(e));
 
-    // holder
-    const existedIdentifierHolder = await this.signifyApi
-      .getIdentifierByName(Agent.HOLDER_AID_NAME)
-      .catch(() => null);
-    if (existedIdentifierHolder) return existedIdentifierHolder;
-
+    // Holder
     await this.signifyApi
       .createIdentifier(Agent.HOLDER_AID_NAME)
       .catch((e) => console.error(e));
     this.keriRegistryRegk = await this.signifyApi
       .createRegistry(Agent.HOLDER_AID_NAME)
       .catch((e) => console.error(e));
+
+    this.createQVICredential().catch((e) => console.error(e));
```

Comment:
`await`

Reply:
If I add wait to this, Server is not running yet => cannot resolveOobi in the issueQVICredential function
![image](https://github.com/cardano-foundation/cf-identity-wallet/assets/142210850/a6344b60-1f15-4a5e-92ed-7453c665dbb3)

Reply:
hmm that's not deterministic then. Can we change it so the server starts up completely before calling `initKeri` then?